packages/react/src/components/tooltip/tooltip.stories.tsx

```diff
@@ -1,11 +1,19 @@
-import { storiesOfTooltip } from "dso-toolkit";
+import type { Meta } from "@storybook/react";
```

Comment:
Deze moet uit `@storybook/web-components` worden geimporteerd.

Reply:
Deze is weer terug uit "@storybook/react"